docs/antrea-network-policy.md

```diff
@@ -1774,3 +1774,5 @@ Similar RBAC is applied to the ClusterGroup resource.
   matched at the same priority with conflicting actions. It will be the policy writer's
   responsibility to identify such ambiguities in rule definitions and avoid potential
   nondeterministic rule enforcement results.
+- NetworPolicy is connection/flow oriented and stateful. It applies to connection, instead of
```

Comment:
If you do not want to start a new paragraph, do not start a new line.

Reply:
to connection -> to connections

Reply:
Fixed. Thanks.
I see the other lines in this file contain no more than 100 characters, thus I follow this behavior for the new paragraph.

Reply:
Do you want to start a new paragraph or you want this new sentence in the previous paragraph?

I meant:
> If you do not want to start a new paragraph, do not start a new line.

Reply:
I want to add one paragraph to explain the NP behavior, and the new paragraph has more than 100 characters, thus I cut the sentence at <100 characters and start a new line as above paragraphs.

Reply:
If you want a new paragraph, you should add an empty new line before the paragraph. Something like:

```
nondeterministic rule enforcement results.

NetworPolicy is connection/flow oriented and stateful. It applies to connection, instead of
```

And we generally wrap lines in docs at width 80, not 100.

Reply:
I reformatted the lines and merged the PR.